swagger_server/controllers_local/collectioninstrument.py

```diff
@@ -0,0 +1,300 @@
+        for testing batches can be initiated with the Swagger UI
+        
+        :param exercise_id: The survey id (this is a uuid)
+        :param count: The number of files we are expecting (in total)
+        :return: At present this routine can't fail as we have no spec for parameter limits
+        """
+        exercise = self._get_exercise(exercise_id)
+        if not exercise:
+            ons_env.session.add(ExerciseModel(exercise_id=exercise_id, items=count))
+            ons_env.session.commit()
+        return 200, {'text': 'OK'}
+
+    @protect(uuid=True)
+    def clear(self, exercise_id):
+        """
+        Clear a batch (or reject a batch) is used to scrap a current batch in the event the user decides they
+        don't actually want to commit it for some reason.
+
+        :param exercise_id: An exercise id, which is a uuid
+        :return: Returns a 200 if the batch was removed, or 204 if not found
+        """
+        exercise = self._get_exercise(exercise_id)
+        if not exercise:
+            return 204, {'text': 'No such batch'}
+        ons_env.session.delete(exercise)
+        ons_env.session.commit()
+        return 200, {'text': 'OK'}
+
+    @protect(uuid=True)
+    def instrument(self, instrument_id):
+        """
+        Return a specific collection instrument based on an id
+
+        :param instrument_id: The id of the instrument we want 
+        :return: formatted JSON version of the instrument
+        """
+        instrument = self._get_instrument(instrument_id)
+        if not instrument:
+            return 404, 'Instrument not found'
+        return 200, instrument.json
+
+    @protect(uuid=True)
+    def status(self, exercise_id):
+        """
+        Get the status of a given batch. This returns a status object containing a number of variables, notably
+        'current' and 'status'.
+
+        :param exercise_id: Survey reference 
+        :return: Return 200 and status object if found, or 204 if batch not found
+        """
+        exercise = self._get_exercise(exercise_id)
+        if not exercise:
+            return 204, {'text': 'No such batch'}
+        return 200, exercise.json
+
+    @protect(uuid=True)
+    def activate(self, exercise_id):
+        """
+        Activate the specified batch and move all associated files into 'active' mode. (database persistence
+        not implemented yet.
+
+        :param exercise_id: Survey reference 
+        :return: 200 if activates, 204 if batch not found, or 500 is batch in wrong state
+        """
+        exercise = self._get_exercise(exercise_id)
+        if not exercise:
+            return 204, {'text': 'No such batch'}
+        if exercise.status != 'pending':
+            return 500, {'text': 'Batch in wrong state'}
+        exercise.status = 'active'
+        ons_env.session.commit()
+        return 200, exercise.json
+
+    @protect(uuid=True)
```

Comment:
You don't necessarily need to set these all to true as the default will handle this